build.gradle.kts

```diff
@@ -79,11 +79,12 @@ project(":api") {
 		implementation("org.jetbrains.kotlinx:kotlinx-coroutines-reactor")
 		testImplementation("io.projectreactor:reactor-test")
 
-		// r2dbc
-		// https://mvnrepository.com/artifact/org.springframework.data/spring-data-r2dbc
-		// TODO please uncomment after setting up r2dbc properties
-		// implementation("org.springframework.data:spring-data-r2dbc:3.0.4")
-		// implementation("dev.miku:r2dbc-mysql:0.8.2.RELEASE")
+		// database
+		runtimeOnly("mysql:mysql-connector-java")
+		implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+		implementation("com.linecorp.kotlin-jdsl:spring-data-kotlin-jdsl-starter:2.0.4.RELEASE")
```

Comment:
It would be nice to have a dependency link as well.

Reply:
Actually, I don't think I need much. But I'll add it in this step.